app/controllers/application_controller.rb

```diff
@@ -2417,9 +2417,7 @@ def set_gettext_locale
                                                       user_settings.key?(:display) &&
                                                       user_settings[:display].key?(:locale)
     if user_locale == 'default' || user_locale.nil?
-      unless MiqServer.my_server.nil?
-        server_locale = MiqServer.my_server.get_config("vmdb").config.fetch_path(:server, :locale)
-      end
+      server_locale = ::Settings.server.locale
```

Comment:
This might actually have a performance improvement as well, since this is called on *every* request.  cc @kbrock